crypto/armcap.c

```diff
@@ -150,6 +151,12 @@ void OPENSSL_cpuid_setup(void)
      * Unified code works because it never triggers SIGILL on Apple
      * devices...
      */
+#   else
+    unsigned int sha512;
+    size_t len = sizeof(sha512);
```

Comment:
We do not allow declarations in the middle of a block. Please enclose this inside `{}`

Reply:
Also there should be an empty line after declarations.